### DISCUSSION TITLE
第二代8G版本在从2.0.7刷到1.6.2时成砖了

### DISCUSSION THREAD

**Discussion by suneao** [PenUniverse/PenMods#221] [你问我答]
在刷2.0.7时岁月静好，就是8G只剩下130MB，于是想要刷到1.6.7
然后，**难受的事情来了**，在刷的途中userdata刷到5%就失败了，然后第二次重刷显示IBM的错误，第三次连**测试设备都过不了**
Loader能连上电脑，但无法写入新系统

**Comment by Redbeanw44602** [comment]
8G版最稳妥的包是网盘里的B-less包，能有1.2G的可用空间。老版本实际上并不能给你带来更多可用空间。

**Reply by suneao** [reply]
B-less包是哪个？能放一下链接吗？
这个版本能装插件、听音乐吗？
B-less是那个2.0.7的吗？不能存储啊

**Reply by laijunyu** [reply]
格式化最后一个分区就能收藏，放歌了

**Reply by suneao** [reply]
最后一个是12吗

**Reply by suneao** [reply]
12格式化不了

**Reply by Huanyi-clr** [reply]
我之前也是，后来装回2.0.7没管
